README.md

```diff
@@ -185,20 +185,20 @@ and/or other countries."
 Thank you to all contributors and a special thanks to:
 
 emeritus reviewers:
-- @ashish-amarnath
-- @davidewatson
-- @enxebre
-- @ingvagabund
-- @michaelbeaumont
-- @sethp-nr
+- [@ashish-amarnath](https://github.com/ashish-amarnath)
+- [@davidewatson](https://github.com/davidewatson)
+- [@enxebre](https://github.com/enxebre)
+- [@ingvagabund](https://github.com/ingvagabund)
+- [@michaelbeaumont](https://github.com/michaelbeaumont)
+- [@sethp-nr](https://github.com/sethp-nr)
 
 emeritus approvers/maintainers:
```

Comment:
Do we need to keep emeritus approvers section on README? I would rather give special thanks to current maintainers, @sedefsavas and @richardcase 😀

If we decide to keep them, let's change `emeritus approvers/maintainers` to `emeritus approvers` as that was what's requested in #3104.

Reply:
Technically, they are maintainers, but also people who where approvers. OWNERS has now the correct format, but missing the right section for these people we invented these new sections so they don't get lost and are still thanked. :)